src/test/ui/rfc-2565-param-attrs/auxiliary/ident-mac.rs

```diff
@@ -0,0 +1,11 @@
+// force-host
+// no-prefer-dynamic
+
+#![crate_type = "proc-macro"]
+
+extern crate proc_macro;
+
+use proc_macro::TokenStream;
+
+#[proc_macro_attribute]
+pub fn id(_: TokenStream, input: TokenStream) -> TokenStream { input }
```

Comment:
We have `auxiliary/test-macros.rs` that defines all the utility macros like this, you can use `#[identity_attr]` from there.

Reply:
I'd have to move the test, right?

Reply:
Ah, ok then.
(We need a way to use a crate from the root `auxiliary` directory in all subdirectories to avoid duplication.)